app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ReviewsController < ApplicationController
+
+  def index
+    @reviews = Review.all
+  end
+
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new({
+      user_id: review_params[:user][:id],
+      content: review_params[:content],
+      film_id: review_params[:filmId]
+      })
```

Comment:
In comments controller your are following the rails convention of nested params, but here you don't do that. You should probably be consistent - use that approach everywhere or nowhere.